solver/pb/ops.proto

```diff
@@ -358,6 +360,13 @@ message FileActionMkFile {
 	int64 timestamp = 5;
 }
 
+message FileActionSymlink {
+	// destination path for the new file representing the link
+	string target = 1;
+	// source path for the link
+	string linkPath = 2;
```

Comment:
just name it `src`.

Alternatively, in Go&unix these are `newpath` and `oldpath`.

Reply:
I just pushed up some updates, but I think I'll modify these to `oldpath` and `newpath`, I think it gives the clearest indication of which is which.

Reply:
done